### PR TITLE
fixes a few bugs/typos and map bugs

### LINKED ISSUE #5181: LV has a double fence on the same tile near a dome entrance
## Testmerges:

N/A

## Reproduction:

Alt-click or Right-click this fence tile and see there are indeed 2. Note you must use Alt-click to attack the second one after the first has been broken in order to pass.

I don't remember exactly where, and the domes move around so don't rely on that for a point of reference sorry.
dreamseeker_zj9nC4AuWy

Also this does not have to do with multi-tile fences if they are even a thing. If you look very closely you can see 2 broken fences in my context menu and only one visible tile out of all the fence you can see is broken.

### LINKED ISSUE #5600: components_base.dm runtime, setup pipenets() related
## Testmerges:

N/A - Master branch 515c6a4 - Automatic changelog generation for PR #5573 [ci skip]

## Log:

My dev server had Prison Station loading at the time, not sure if it is map specific.

Update: Splinter reports it is a Prison specific issue and could have to do with modular mapping in this area
image

[22:47:55] Runtime in components_base.dm, line 89: list index out of bounds
```
proc name: setPipenet (/obj/machinery/atmospherics/components/setPipenet)
src: the air scrubber (/obj/machinery/atmospherics/components/unary/vent_scrubber)
src.loc: the floor (34,91,2) (/turf/open/floor/prison/bright_clean/two)
call stack:
the air scrubber (/obj/machinery/atmospherics/components/unary/vent_scrubber): setPipenet(/datum/pipeline (/datum/pipeline), the scrubbers pipe (/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden))
/datum/pipeline (/datum/pipeline): build pipeline(the scrubbers pipe (/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden))
the scrubbers pipe (/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden): build network()
Atmospherics (/datum/controller/subsystem/air): setup pipenets()
Atmospherics (/datum/controller/subsystem/air): Initialize(244749)
Master (/datum/controller/master): Initialize(10, 0, 1)
```

### LINKED ISSUE #5603: not wearing gloves makes you immune to hallucinatory door shocks
```/obj/machinery/door/airlock/bumpopen(mob/living/user) //Airlocks now zap you when you 'bump' them open when they're electrified. --NeoFite
    if(issilicon(user))
        return ..(user)
    if(iscarbon(user) && isElectrified())
        if(!justzap)
            if(shock(user, 100))
                justzap = TRUE
                spawn (openspeed)
                    justzap = FALSE
                return
        else /*if(justzap)*/
            return
    else if(ishuman(user) && user.hallucination > 50 && prob(10) && !operating)
        var/mob/living/carbon/human/H = user
        if(H.gloves)
            to_chat(H, "<span class='danger'>You feel a powerful shock course through your body!</span>")
            var/obj/item/clothing/gloves/G = H.gloves
            if(G.siemens_coefficient)//not insulated
                H.adjustStaminaLoss(200)
                return
    return ..(user)

### LINKED ISSUE #5605: the code for honey's healing effects uses some commas where it really should be using periods
```
/datum/reagent/consumable/honey
    name = "Honey"
    description = "A golden yellow syrup, loaded with sugary sweetness."
    color = "#FFFF00"
    nutriment_factor = 15
    taste_description = "sweetness"

/datum/reagent/consumable/honey/on_mob_life(mob/living/L, metabolism)
    L.reagents.add_reagent(/datum/reagent/consumable/sugar,3)
    L.adjustBruteLoss(-0,5 * REM)
    L.adjustFireLoss(-0,5 * REM)
    L.adjustOxyLoss(-0,5 * REM)
    L.adjustToxLoss(-0,5 * REM)
    return ..()
```
I believe that the game will interpret this as an attempt to pass an argument of "-0" and an argument of "5 * REM" instead of as an attempt to pass a single argument of "-0.5 * REM".

### PR DESCRIPTION
**Pull request by SplinterGP**
## About The Pull Request
Fixes #5603 
Fixes #5181 
Fixes #5600 
Fixes #5605 

also fixes a rogue continue on explosion code

## Why It's Good For The Game
Fixes bugs

## Changelog
:cl:
fix: fixes code of honey where it should use period instead of comma.
fix: fixes runtime at prison station init.
fix: fixes hallucination shocking not working when you don't have gloves.
fix: fixes rogue return at explosions code.
fix: fixes a spot of LV where it had two fences at same tile
/:cl:

### CODE REVIEW

File: code/controllers/subsystem/explosions.dm [hunk 1]
```diff
@@ -207,7 +207,7 @@ SUBSYSTEM_DEF(explosions)
 	for(var/obj/blocking_object in epicenter)
 		if(!blocking_object.density)
 			continue
-			current_exp_block += ( (blocking_object.explosion_block == EXPLOSION_BLOCK_PROC) ? blocking_object.GetExplosionBlock(0) : blocking_object.explosion_block ) //0 is the result of get_dir between two atoms on the same tile.
+		current_exp_block += ( (blocking_object.explosion_block == EXPLOSION_BLOCK_PROC) ? blocking_object.GetExplosionBlock(0) : blocking_object.explosion_block ) //0 is the result of get_dir between two atoms on the same tile.
```
Comment (TiviPlus) [inline code/controllers/subsystem/explosions.dm:210]: Fix the one in admin drawing tools too and report the bug on Spaceman DMM please

Comment (SplinterGP) [reply code/controllers/subsystem/explosions.dm:210]: where's that ?

**Comment by SplinterGP** [comment]
oh wtf, commit garbages, bleh

**Comment by SplinterGP** [comment]
fuk git

**Comment by SplinterGP** [comment]
gonna re-open PR, Github dying on it

**Pull request closed without merging**